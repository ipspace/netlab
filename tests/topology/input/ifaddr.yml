defaults:
  device: cumulus
  
nodes:
  r1:
    id: 17
  r2:
    id: 42
  r3:
    id: 50

links:
- r1:
  r2:
  prefix:
    ipv4: 192.168.1.0/24
    ipv6: 2001:db8:0:42::/64
  type: lan
- type: lan
  r1:
    ipv4: 1.2.3.5
  r2:
    ipv4: 1.2.3.4/29
    ipv6: 2001:db8:1:1::1/127
  r3:
