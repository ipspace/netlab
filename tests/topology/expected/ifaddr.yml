input:
- topology/input/ifaddr.yml
- package:topology-defaults.yml
links:
- bridge: input_1
  linkindex: 1
  node_count: 2
  prefix:
    ipv4: 192.168.1.0/24
    ipv6: 2001:db8:0:42::/64
  r1:
    ipv4: 192.168.1.17/24
    ipv6: 2001:db8:0:42::11/64
  r2:
    ipv4: 192.168.1.42/24
    ipv6: 2001:db8:0:42::2a/64
  type: lan
- bridge: input_2
  linkindex: 2
  node_count: 3
  prefix:
    ipv4: 172.16.0.0/24
  r1:
    ipv4: 1.2.3.5/24
  r2:
    ipv4: 1.2.3.4/29
    ipv6: 2001:db8:1:1::1/127
  r3:
    ipv4: 172.16.0.50/24
  type: lan
name: input
nodes:
- box: CumulusCommunity/cumulus-vx
  device: cumulus
  id: 17
  links:
  - bridge: input_1
    ifindex: 1
    ifname: swp1
    ipv4: 192.168.1.17/24
    ipv6: 2001:db8:0:42::11/64
    linkindex: 1
    name: r1 -> [r2]
    neighbors:
      r2:
        ifname: swp1
        ipv4: 192.168.1.42/24
        ipv6: 2001:db8:0:42::2a/64
    type: lan
  - bridge: input_2
    ifindex: 2
    ifname: swp2
    ipv4: 1.2.3.5/24
    linkindex: 2
    name: r1 -> [r2,r3]
    neighbors:
      r2:
        ifname: swp2
        ipv4: 1.2.3.4/29
        ipv6: 2001:db8:1:1::1/127
      r3:
        ifname: swp1
        ipv4: 172.16.0.50/24
    type: lan
  loopback:
    ipv4: 10.0.0.17/32
  mgmt:
    ifname: eth0
    ipv4: 192.168.121.117
    mac: 08-4F-A9-00-00-11
  name: r1
- box: CumulusCommunity/cumulus-vx
  device: cumulus
  id: 42
  links:
  - bridge: input_1
    ifindex: 1
    ifname: swp1
    ipv4: 192.168.1.42/24
    ipv6: 2001:db8:0:42::2a/64
    linkindex: 1
    name: r2 -> [r1]
    neighbors:
      r1:
        ifname: swp1
        ipv4: 192.168.1.17/24
        ipv6: 2001:db8:0:42::11/64
    type: lan
  - bridge: input_2
    ifindex: 2
    ifname: swp2
    ipv4: 1.2.3.4/29
    ipv6: 2001:db8:1:1::1/127
    linkindex: 2
    name: r2 -> [r1,r3]
    neighbors:
      r1:
        ifname: swp2
        ipv4: 1.2.3.5/24
      r3:
        ifname: swp1
        ipv4: 172.16.0.50/24
    type: lan
  loopback:
    ipv4: 10.0.0.42/32
  mgmt:
    ifname: eth0
    ipv4: 192.168.121.142
    mac: 08-4F-A9-00-00-2A
  name: r2
- box: CumulusCommunity/cumulus-vx
  device: cumulus
  id: 50
  links:
  - bridge: input_2
    ifindex: 1
    ifname: swp1
    ipv4: 172.16.0.50/24
    linkindex: 2
    name: r3 -> [r1,r2]
    neighbors:
      r1:
        ifname: swp2
        ipv4: 1.2.3.5/24
      r2:
        ifname: swp2
        ipv4: 1.2.3.4/29
        ipv6: 2001:db8:1:1::1/127
    type: lan
  loopback:
    ipv4: 10.0.0.50/32
  mgmt:
    ifname: eth0
    ipv4: 192.168.121.150
    mac: 08-4F-A9-00-00-32
  name: r3
provider: libvirt
